test/index.js

```diff
@@ -1,24 +1,32 @@
-const query = require('../');
-const { expect } = require('chai');
+const query = require("../");
```

Comment:
I have prettier installed in my-editor and it changed the quotes and maybe other things here.
If that's not ok, I'll revert those changes.